bbc_clean/treerez/README.md

```diff
@@ -0,0 +1,26 @@
+# TreeRez
```

Comment:
Maybe you could add the reason why we needed to develop this tool in the first place.